src/commands/publish/mod.rs

```diff
@@ -43,46 +36,10 @@ pub fn publish(user: &GlobalUser, project: &Project, release: bool) -> Result<()
     Ok(())
 }
 
-pub fn create_kv_namespaces(user: &GlobalUser, project: &Project) -> Result<(), failure::Error> {
-    let kv_addr = format!(
-        "https://api.cloudflare.com/client/v4/accounts/{}/storage/kv/namespaces",
-        project.account_id,
-    );
-
-    let client = http::auth_client(user);
-
-    if let Some(namespaces) = &project.kv_namespaces {
-        for namespace in namespaces {
-            info!("Attempting to create namespace '{}'", namespace);
-
-            let mut map = HashMap::new();
-            map.insert("title", namespace);
-
-            let request = client.post(&kv_addr).json(&map).send();
-
-            if let Err(error) = request {
-                // A 400 is returned if the account already owns a namespace with this title.
-                //
-                // https://api.cloudflare.com/#workers-kv-namespace-create-a-namespace
-                match error.status() {
-                    Some(code) if code == 400 => {
-                        info!("Namespace '{}' already exists, continuing.", namespace)
-                    }
-                    _ => {
-                        info!("Error when creating namespace '{}'", namespace);
-                        failure::bail!("⛔ Something went wrong! Error: {}", error)
-                    }
-                }
-            }
-            info!("Namespace '{}' exists now", namespace)
-        }
-    }
-    Ok(())
-}
-
-fn publish_script(
+fn publish_worker(
```

Comment:
we don't ever publish anything that is not a worker- publish vs publish worker here is not clear

Reply:
Would `publish_bundle` be better?